static/js/portico/header.js

```diff
@@ -13,4 +13,10 @@ $(function () {
             $(".dropdown").removeClass("show");
         }
     });
+
+    if ($(".desktop-redirect-links").length > 0) {
+        setTimeout(() => {
+            document.getElementById('desktop-redirect-link').click();
+        }, 3000);
```

Comment:
Perhaps 3 seconds are a bit too long of a delay?

Reply:
Will change to 2 seconds then ? 1 second is very fast, acc. to me.